tokens/eoskingdom/EKD.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "EKD",
+  "symbol": "EKD",
+  "contract": "ekdtokenbank",
```

Comment:
Please detailed why this token is uploaded under eoskingdom while its contract is ekdtokenbank.